src/course-unit/course-xblock/CourseXBlock.jsx

```diff
@@ -76,12 +76,17 @@ const CourseXBlock = ({
 
   return (
     <div ref={courseXBlockElementRef} {...props}>
-      <Card as={ConditionalSortableElement} id={id} draggable className="mb-1">
+      <Card
+        as={ConditionalSortableElement}
+        id={id}
+        draggable
+        componentStyle={{ marginBottom: 0 }}
```

Comment:
Will `className=""mb-0` work. It is preferable to avoid inline styles

Reply:
Unfortunately no